packages/teleport/src/config.ts

```diff
@@ -55,6 +56,12 @@ const cfg = {
 
   proxyCluster: 'localhost',
 
+  scheduledUpgradeWindows: [
+    '08:00:00',
+    '16:00:00',
+    '23:00:00',
+  ] as UpgradeWindowStart[],
```

Comment:
im not sure this is the right place, these are just consts and i would move and define these consts to the component that it is being used